lib/ansible/modules/cloud/amazon/lambda_policy.py

```diff
@@ -15,8 +15,7 @@
     - This module allows the management of AWS Lambda policy statements.
       It is idempotent and supports "Check" mode.  Use module M(lambda) to manage the lambda
       function itself, M(lambda_alias) to manage function aliases, M(lambda_event) to manage event source mappings
-      such as Kinesis streams, M(lambda_invoke) to execute a lambda function and M(lambda_facts) to gather facts
-      relating to one or more lambda functions.
+      such as Kinesis streams and M(lambda_facts) to gather facts relating to one or more lambda functions.
```

Comment:
It seems lambda_invoke isn't, but execute_lambda (same functionality) is part of Ansible since 2.2

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/execute_lambda.py